wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -864,6 +864,12 @@ extension AuthenticationCoordinator {
                         .transition(.enrollE2EIdentitySuccess(certificateChain), mode: .reset)
                     ])
                 }
+            } catch OAuthError.userCancelled {
+                await MainActor.run {
+                    executeActions([
+                        .hideLoadingView
+                    ])
+                }
```

Comment:
It would be better to mark `func executeActions` with `@MainActor` instead as it does UI operations and then the compiler helps us to find issues.